oaknational/Oak-Web-Application
# On boarding page

## Description

- Adds onboarding (splash) page. [figma](https://www.figma.com/file/96yUIKNRvg0rMCWasdkJ1u/Design-System-2.0?node-id=1939%3A17077)
- Links from landing page to on boarding
- Card links to pupil and teacher home pages (Images not included on card until we have suitable placeholders)
- Adds teacher and pupil home pages

## Issue(s)

https://github.com/oaknational/Samara/issues/238

Fixes #238 

## How to test

1. Go to {deployment link}
2. Click on use Oak in beta card
3. You should see on boarding page (placeholder blank pages with title)

## Screenshots

## Checklist
